src/Features/Core/Portable/GenerateMember/GenerateVariable/AbstractGenerateVariableService.cs

```diff
@@ -49,29 +50,25 @@ protected AbstractGenerateVariableService()
 
                 var canGenerateMember = CodeGenerator.CanAdd(document.Project.Solution, state.TypeToGenerateIn, cancellationToken);
 
-                // prefer fields over properties (and vice versa) depending on the casing of the member.
-                // lowercase -> fields.  title case -> properties.
-                var name = state.IdentifierToken.ValueText;
-                if (char.IsUpper(name.ToCharArray().FirstOrDefault()))
+                if (canGenerateMember)
                 {
-                    if (canGenerateMember)
+                    // prefer fields over properties (and vice versa) depending on the casing of the member.
+                    // lowercase -> fields.  title case -> properties.
+                    var name = state.IdentifierToken.ValueText;
+                    if (char.IsUpper(name.ToCharArray().FirstOrDefault()))
                     {
                         AddPropertyCodeActions(actions, semanticDocument, state);
                         AddFieldCodeActions(actions, semanticDocument, state);
                     }
-
-                    AddLocalCodeActions(actions, document, state);
-                }
-                else
-                {
-                    if (canGenerateMember)
+                    else
                     {
+
                         AddFieldCodeActions(actions, semanticDocument, state);
                         AddPropertyCodeActions(actions, semanticDocument, state);
                     }
-
-                    AddLocalCodeActions(actions, document, state);
                 }
+                AddLocalCodeActions(actions, document, state);
+                AddParameterCodeActions(actions, document, state);
```

Comment:
i think i've convinced myself this has no change to existing funcitonality, except to add the parameter option if applicable.  can you verify?

Reply:
Yep. I refactored it slightly to avoid duplicating the call to `AddParameterCodeActions`